packages/react/src/components/VideoPlayer/__stories__/VideoPlayer.stories.js

```diff
@@ -51,3 +38,53 @@ export const Default = ({ parameters }) => {
     </div>
   );
 };
+
+Default.story = {
+  parameters: {
+    knobs: {
+      VideoPlayer: ({ groupId }) => ({
+        showCaption: boolean('Show caption (showCaption):', true, groupId),
+        aspectRatio: 'default',
+        videoId: '0_uka1msg4',
```

Comment:
@guilhermelMoraes for default we should also use a media center video and not the kaltura public video (folgers)